ecosystem/sep-0006.md

```diff
@@ -838,6 +839,36 @@ For example:
 }
 ```
 
+## Wallet Attribution
+
+Using [SEP-34](sep-0034.md), wallets can prove their identity to anchors for requests to process a particular transaction. This endpoint defines how the wallet should send the JWS token generated and returned from the wallet's signing server to the anchor service.
+
+### Request
+
+```
+POST TRANSFER_SERVER/transaction/attribution
+```
+
+Request parameters:
+
+Name | Type | Description
+-----|------|------------
+`id` | string | The id of the transaction.
+`token` | string | The JWS token returned by the wallet's [signing server](sep-0034.md)
+
+See [Verifying Requests](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0034.md#verifying-requests) for more information about validating the tokens passed.
+
+### Response
```

Comment:
If the request is valid but the wallet has has already sent on-chain funds to the anchor or the user has sent off-chain funds to the anchor, the anchor shouldn't process the request. Transaction that can be attributed to clients must be in the `incomplete` status, meaning the user or client has not been tasked with sending funds to the anchor.

Anchors should return a `422 Unprocessable Entity` status, meaning the anchor does not refund fees paid that were attributed to a wallet discount or sponsorship policy after the transaction funds were sent and fees paid.